src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -376,6 +376,16 @@ private static int LineBreaksAfterCloseBrace(SyntaxToken currentToken, SyntaxTok
                 return 0;
             }
 
+            // If we are at the end of a property followed by another property
+            // group them together by having only 1 line break.
+            // The current token here is a closing brace of an accessor list:
+            // public int Prop { get; } <-- this one
```

Comment:
note: i would only expect this for props that themselves are only on a single line.  multi-line props would continue to be separated by a blank line.